### PR TITLE
Server Implementation

### PR DESCRIPTION
**Pull request by abharun**

### CODE REVIEW

File: server/api/animal_img/services.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import requests
+import base64
+from dotenv import load_dotenv
+from consts.urls import IMAGE_URL
+import os
+import time
+
+load_dotenv()
```
Comment (FreeFly0125) [inline server/api/animal_img/services.py:8]: we can remove this as we are not using env values.

File: server/images/img.png
Comment (FreeFly0125) [inline server/images/img.png:1]: just add to .gitignore

**Review by FreeFly0125 — CHANGES_REQUESTED** [review]

File: server/api/animal_img/services.py (hunk 1 above)
Comment (abharun) [reply server/api/animal_img/services.py:8]: sure.

File: server/images/img.png
Comment (abharun) [reply server/images/img.png:1]: fixed.

**Review by FreeFly0125 — APPROVED** [review]

**Pull request merged**